### PR TITLE
store disk type and performance level as slice

### PR DESCRIPTION
**Pull request by huww98**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Avoid repeated split and join.

#### Which issue(s) this PR fixes:

The original `validateDiskPerformaceLevel` does not validate the input if it does not contains comma. Fixed this.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/disk/utils.go [hunk 1]
```diff
@@ -1036,33 +1036,32 @@ func validateDiskType(opts map[string]string) (diskType string, err error) {
...
 	log.Infof("validateDiskPerformanceLevel: pl: %v", pl)
-	if strings.Contains(pl, ",") {
-		for _, cusPer := range strings.Split(pl, ",") {
-			if _, ok := CustomDiskPerfermance[cusPer]; !ok {
-				return "", fmt.Errorf("illegal performance level type: %s", cusPer)
-			}
+	performanceLevel = strings.Split(pl, ",")
+	for _, cusPer := range performanceLevel {
+		if _, ok := CustomDiskPerfermance[cusPer]; !ok {
+			return nil, fmt.Errorf("illegal performance level type: %s", cusPer)
```
Comment (mowangdk) [inline pkg/disk/utils.go:1061]: return default value ```[]string{}``` instead of nil?

Comment (huww98) [reply pkg/disk/utils.go:1061]: The default value of []string is nil.

**Pull request merged**